fish_prompt.fish

```diff
@@ -449,7 +451,11 @@ function __bobthefish_prompt_status -S -a last_status -d 'Display flags for a no
         if [ "$bg_jobs" ]
             set_color normal
             set_color -b $color_initial_segment_jobs
-            echo -n $bg_job_glyph
+            if [ "$theme_show_job_number" = 'yes' ]
+                echo -ns $bg_job_glyph $bg_jobs ' '
```

Comment:
you called this variable `$num_bg_jobs` :P 

… but with my change above, we can leave it `$bg_jobs`.

Reply:
I'll use your version with the above code so that we can leave it `$bg_jobs`